packages/local-relay/src/RelayServer.ts

```diff
@@ -0,0 +1,220 @@
+import express from 'express';
+import * as websocket from 'websocket';
+import * as http from 'http';
+import * as net from 'net';
+
+import HostStore from './HostStore';
+import Host from './Host';
+
+type RequestResponse = {
+  status: number;
+};
+
+export default class RelayServer {
+  private app = express();
+  public readonly httpServer: http.Server;
+  public readonly websocketServer: websocket.server;
+
+  private hostStore = new HostStore();
+
+  constructor() {
+    this.app.disable('x-powered-by');
+
+    this.httpServer = http.createServer(this.app);
+    this.websocketServer = new websocket.server({
+      httpServer: this.httpServer,
+    });
+
+    this.app
+      .disable('x-powered-by')
+      .get('/', this.onAppGetRoot)
+      .get('/1/user/-/developer-relay/hosts.json', this.lookupHosts.bind(this))
+      .post(
+        '/1/user/-/developer-relay/hosts/:id',
+        this.createConnectionURL.bind(this),
+      )
+      .delete('/connections/:connectionId', this.closeConnection.bind(this));
```

Comment:
I don't think this is necessary in a local relay, it's intended for cross-relay-node connection cleanup.